src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -83,6 +83,7 @@ private static ModElement me(Workspace workspace, ModElementType type, String su
 			generatableElements.add(getToolExample(me(workspace, type, "9"), "Special", random, true, false));
 			generatableElements.add(getToolExample(me(workspace, type, "10"), "MultiTool", random, true, false));
 			generatableElements.add(getToolExample(me(workspace, type, "11"), "Shears", random, true, false));
+			generatableElements.add(getToolExample(me(workspace, type, "12"), "Shield", random, true, false));

```

Comment:
You will need to specify the provider in `getToolExample`, just calling a tool type that does not exist here is not enough.

Reply:
What do you mean by that? How do I specify the provider?